keras/meta.yaml

```diff
@@ -27,14 +27,20 @@ requirements:
     - python
     - pyyaml
     - keras-preprocessing  >=1.1.2
+  run_constrained:
```

Comment:
I'm not sure about this one. The documentation says that run_constrained packages are optional. But keras won't work without tensorflow.